src/lib/services/access-service.ts

```diff
@@ -275,7 +275,9 @@ export class AccessService {
 
     async getUserRootRoles(userId: number): Promise<IRoleWithProject[]> {
         const userRoles = await this.store.getRolesForUserId(userId);
-        return userRoles.filter((r) => r.type === RoleType.ROOT);
+        return userRoles.filter(
+            (r) => r.type === RoleType.ROOT || r.type === RoleType.ROOT_CUSTOM,
```

Comment:
This is the fix

Reply:
Maybe consider `({type}) => ROOT_ROLE_TYPES.includes(type)` in case it's cleaner.

Reply:
Addressed in [9956e9e](https://github.com/Unleash/unleash/pull/4068/commits/9956e9e9caf3940b1eefcad3c903593b7ead9804)